wolfcrypt/src/ecc.c

```diff
@@ -4855,10 +4858,26 @@ static int ecc_make_pub_ex(ecc_key* key, ecc_curve_spec* curveIn,
     #endif
     }
 
+#if defined(WOLFSSL_ASYNC_CRYPT) && defined(WC_ASYNC_ENABLE_ECC) && \
+    defined(HAVE_INTEL_QA)
+    if (err == MP_OKAY) {
+        word32 keySz = key->dp->size;
+        /* sync private key to raw */
+        err = wc_mp_to_bigint_sz(&key->k, &key->k.raw, keySz);
+        if (err == MP_OKAY) {
+            err = IntelQaEccPointMul(&key->asyncDev,
+                &key->k.raw, pub->x, pub->y, pub->z,
```

Comment:
Should this be pub->x->raw?

Reply:
For the ECC key gen I am passing in the mp_int and populating directly in the Intel QAT callback. The NUMA raw is allocated in the quickassist.c instead. This allowed implementation without having to setup states in key gen and make pub.